### ISSUE TITLE
`UnicodeEncodeError: 'cp932' codec can't encode character '\U0001f31f' in position 259: illegal multibyte sequence` occurs after v0.184.0.

### ISSUE THREAD

**Issue by juner** [microsoft/vscode-remote-release#5308]
`UnicodeEncodeError: 'cp932' codec can't encode character '\U0001f31f' in position 259: illegal multibyte sequence` occurs after v0.184.0.  (* I also confirmed it in v0.187.0 )
- VSCode Version: 1.58.0
- Local OS Version: Windows10 21H1 (19043.1081)
- Windows System Locale: ja-JP (1041)
- Remote Extension: Docker
  - Remote - Containers (`ms-vscode-remote.remote-containers`): v0.184.0
- Logs: 
<details><summary>detail</summary>

```log
[2021-07-09T08:26:12.988Z] Remote-Containers 0.184.0 in VS Code 1.58.0 (2d23c42a936db1c7b3b06f918cde29561cc47cd6).
[2021-07-09T08:26:12.988Z] Start: Resolving Remote
[2021-07-09T08:26:12.992Z] Setting up container for folder or workspace: ************************************\errord_devcontainer20210709
[2021-07-09T08:26:12.994Z] Start: Check Docker is running
[2021-07-09T08:26:12.995Z] Start: Run: docker version --format {{.Server.APIVersion}}
[2021-07-09T08:26:13.471Z] Stop (476 ms): Run: docker version --format {{.Server.APIVersion}}
[2021-07-09T08:26:13.471Z] Server API version: 1.41
[2021-07-09T08:26:13.471Z] Stop (477 ms): Check Docker is running
[2021-07-09T08:26:13.580Z] Start: Run: docker-compose version --short
[2021-07-09T08:26:14.786Z] Stop (1206 ms): Run: docker-compose version --short
[2021-07-09T08:26:14.788Z] Start: Run: docker ps -q -a --filter label=com.docker.compose.project=errord_devcontainer20210709 --filter label=com.docker.compose.service=php
[2021-07-09T08:26:15.309Z] Stop (521 ms): Run: docker ps -q -a --filter label=com.docker.compose.project=errord_devcontainer20210709 --filter label=com.docker.compose.service=php
[2021-07-09T08:26:15.310Z] Start: Run: docker inspect --type container b2e695a99e08
[2021-07-09T08:26:15.807Z] Stop (497 ms): Run: docker inspect --type container b2e695a99e08
[2021-07-09T08:26:15.809Z] Start: Run: docker-compose -f ************************************\errord_devcontainer20210709\docker-compose.yml config
[2021-07-09T08:26:17.530Z] Stop (1721 ms): Run: docker-compose -f ************************************\errord_devcontainer20210709\docker-compose.yml config
[2021-07-09T08:26:17.530Z] 
[2021-07-09T08:26:17.530Z] Traceback (most recent call last):
  File "docker-compose", line 3, in <module>
  File "compose\cli\main.py", line 81, in main
  File "compose\cli\main.py", line 197, in perform_command
  File "compose\metrics\decorator.py", line 18, in wrapper
  File "compose\cli\main.py", line 441, in config
UnicodeEncodeError: 'cp932' codec can't encode character '\U0001f31f' in position 259: illegal multibyte sequence
[6368] Failed to execute script docker-compose

[2021-07-09T08:26:17.530Z] Exit code 4294967295
[2021-07-09T08:26:17.534Z] Command failed: docker-compose -f ************************************\errord_devcontainer20210709\docker-compose.yml config

```

</details>

Steps to Reproduce:

1.  use project [errord_devcontainer20210709](https://github.com/juner/errord_devcontainer20210709)
2. Windows System Locale is `ja-JP`
3. Open the target project with `Open Folder in Container`. 

* This did not occur with `v0.183.0`. 

* Does this issue occur when you try this locally?: `Yes`
* Does this issue occur when you try this locally and all extensions are disabled?: `Yes`

**Comment by juner** [comment]
This issue was reproduced even if `Windows System Locale` was` en-US`.

**Comment by jkklapp** [comment]
I can reproduce this issue in mac os
```
[295 ms] Remote-Containers 0.187.0 in VS Code 1.58.1 (2aeda6b18e13c4f4f9edf6667158a6b8d408874b).
[295 ms] Start: Resolving Remote
[298 ms] Setting up container for folder or workspace: ***********************************************.code-workspace
[311 ms] Start: Check Docker is running
[311 ms] Start: Run: docker version --format {{.Server.APIVersion}}
[650 ms] Server API version: 1.41
[670 ms] Start: Run: docker-compose version --short
[1464 ms] Start: Run: docker ps -q -a --filter label=com.docker.compose.project=* --filter label=com.docker.compose.service=api
[1775 ms] Start: Run: docker-compose -f *********************************************/docker-compose.yml config
[2367 ms] 
[2367 ms] Python-dotenv could not parse statement starting at line 55
Traceback (most recent call last):
  File "docker-compose", line 3, in <module>
  File "compose/cli/main.py", line 81, in main
  File "compose/cli/main.py", line 197, in perform_command
  File "compose/metrics/decorator.py", line 18, in wrapper
  File "compose/cli/main.py", line 441, in config
UnicodeEncodeError: 'ascii' codec can't encode character '\u201c' in position 2030: ordinal not in range(128)
[37346] Failed to execute script docker-compose

[2367 ms] Exit code 255
[2371 ms] Command failed: docker-compose -f ********************************************/docker-compose.yml config
```

**Comment by chrmarti** [comment]
We used to run `docker-compose -f ***/docker-compose.yml config --services` and then dropped `--services` in 0.184.0 because we sometimes need to inspect the full configuration.

Can you run `docker-compose -f ***/docker-compose.yml config` from the command line?

**Comment by juner** [comment]
@chrisdias 

```pwsh
PS ***********> docker-compose -f c:\************************\errord_devcontainer20210709\docker-compose.yml config
services:
  php:
    build:
      args:
        INSTALL_NODE: "true"
        NODE_VERSION: lts/*
        VARIANT: '7'
      context: c:\************************\errord_devcontainer20210709\.container
    environment:
      APP_TITLE: ＰＨＰサンプル🌟
    volumes:
    - c:\************************\errord_devcontainer20210709:/var/www/html:rw
version: '3.7'

PS C:\***********>
```

**Comment by jkklapp** [comment]
FYI, rolling back to 0.183.0 fix the problem

**Comment by chrmarti** [comment]
Including a workaround in 0.188.2 (currently requires VS Code Insiders). Could both of you seeing this verify the workaround in that version? I'm considering to also it for VS Code 1.58.

**Comment by juner** [comment]
It worked. 

<details><summary>logs</summary>

```
[1465 ms] Remote-Containers 0.188.2 in VS Code 1.59.0-insider (d624242076d166bb8ce8b314e8d44b8af6100a3d).
[1464 ms] Start: Resolving Remote
[1498 ms] Setting up container for folder or workspace: c:\Users\juner\source\repos\errord_devcontainer20210709
[1504 ms] Start: Check Docker is running
[1504 ms] Start: Run: docker version --format {{.Server.APIVersion}}
[2836 ms] Server API version: 1.41
[3118 ms] Start: Run: docker-compose version --short
[6658 ms] Start: Run: docker ps -q -a --filter label=com.docker.compose.project=errord_devcontainer20210709 --filter label=com.docker.compose.service=php
[7436 ms] Start: Run: docker-compose -f c:\Users\juner\source\repos\errord_devcontainer20210709\docker-compose.yml config
[9712 ms] 
[9712 ms] Traceback (most recent call last):
  File "docker-compose", line 3, in <module>
  File "compose\cli\main.py", line 81, in main
  File "compose\cli\main.py", line 197, in perform_command
  File "compose\metrics\decorator.py", line 18, in wrapper
  File "compose\cli\main.py", line 441, in config
UnicodeEncodeError: 'cp932' codec can't encode character '\U0001f31f' in position 250: illegal multibyte sequence
[3176] Failed to execute script docker-compose

[9712 ms] Exit code 4294967295
[9713 ms] Start: Run: docker-compose -f c:\Users\juner\source\repos\errord_devcontainer20210709\docker-compose.yml config
services:
  php:
    build:
      args:
        INSTALL_NODE: "true"
        NODE_VERSION: lts/*
        VARIANT: '7'
      context: c:\Users\juner\source\repos\errord_devcontainer20210709\.container
    environment:
      APP_TITLE: ＰＨＰサンプル�
    volumes:
    - c:\Users\juner\source\repos\errord_devcontainer20210709:/var/www/html:rw
version: '3.7'

[12558 ms] Start: Run: docker events --format {{json .}} --filter event=start
[12608 ms] Start: Run: docker-compose --project-name errord_devcontainer20210709 -f c:\Users\juner\source\repos\errord_devcontainer20210709\docker-compose.yml build
Building php
[+] Building 1.7s (6/6) FINISHED
 => [internal] load build definition from Dockerfile                                 0.1s
 => => transferring dockerfile: 32B                                                  0.0s
 => [internal] load .dockerignore                                                    0.1s
 => => transferring context: 2B                                                      0.0s
 => [internal] load metadata for mcr.microsoft.com/vscode/devcontainers/php:0-7      1.3s
 => [1/2] FROM mcr.microsoft.com/vscode/devcontainers/php:0-7@sha256:b6f999ddc37961  0.0s
 => CACHED [2/2] RUN if [ "true" = "true" ]; then su vscode -c "umask 0002 && . /us  0.0s
 => exporting to image                                                               0.2s
 => => exporting layers                                                              0.0s
 => => writing image sha256:dafaf818a78c73b3baaaa4076aba5a367b313901c246e74e7246bba  0.0s
 => => naming to docker.io/library/errord_devcontainer20210709_php                   0.0s

Use 'docker scan' to run Snyk tests against images to find vulnerabilities and learn how t
o fix them
[20728 ms] Start: Run: docker-compose --project-name errord_devcontainer20210709 -f c:\Users\juner\source\repos\errord_devcontainer20210709\docker-compose.yml up -d
Creating errord_devcontainer20210709_php_1 ... done
[25536 ms] Start: Run: docker ps -q -a --filter label=com.docker.compose.project=errord_devcontainer20210709 --filter label=com.docker.compose.service=php
[26183 ms] Start: Run: docker inspect --type container 36f3bf0f05d7
[26653 ms] Start: Inspecting container
[26653 ms] Start: Run: docker inspect --type container 36f3bf0f05d7d853bdc88269244301e9582071f6f39f9d68d0c0290e8f7ba057
[27174 ms] Start: Run in container: /bin/sh
[27209 ms] Start: Run in container: uname -m
[27925 ms] x86_64
[27926 ms] 
[27926 ms] Start: Run in container: (cat /etc/os-release || cat /usr/lib/os-release) 2>/dev/null
[27933 ms] PRETTY_NAME="Debian GNU/Linux 10 (buster)"
NAME="Debian GNU/Linux"
VERSION_ID="10"
VERSION="10 (buster)"
VERSION_CODENAME=buster
ID=debian
HOME_URL="https://www.debian.org/"
SUPPORT_URL="https://www.debian.org/support"
BUG_REPORT_URL="https://bugs.debian.org/"
[27934 ms] 
[27934 ms] Start: Run in container: cat /etc/passwd
[27938 ms] Start: Updating configuration state
[28013 ms] Start: Setup shutdown monitor
[28014 ms] Forking shutdown monitor: c:\Users\juner\.vscode-insiders\extensions\ms-vscode-remote.remote-containers-0.188.2\dist\shutdown\shutdownMonitorProcess \\.\pipe\vscode-remote-containers-ed35a5002b570dd1cc37a29ef77deda86a8d4d13-sock dockerCompose Debug c:\Users\juner\AppData\Roaming\Code - Insiders\logs\20210720T020325\exthost1\ms-vscode-remote.remote-containers 1626714211782
[28033 ms] Start: Run in container: test -d /home/vscode/.vscode-server-insiders
[28041 ms] 
[28042 ms] 
[28042 ms] Exit code 1
[28043 ms] Start: Run in container: test -d /home/vscode/.vscode-remote
[28048 ms] 
[28049 ms] 
[28049 ms] Exit code 1
[28049 ms] Start: Run in container: test -f /var/vscode-server-insiders/.patchEtcEnvironmentMarker
[28054 ms] 
[28054 ms] 
[28054 ms] Exit code 1
[28055 ms] Start: Run in container: /bin/sh
[28098 ms] Start: Run in container: test ! -f '/var/vscode-server-insiders/.patchEtcEnvironmentMarker' && set -o noclobber && mkdir -p '/var/vscode-server-insiders' && { > '/var/vscode-server-insiders/.patchEtcEnvironmentMarker' ; } 2> /dev/null
[29408 ms] 
[29408 ms] 
[29409 ms] Start: Run in container: cat >> /etc/environment <<'etcEnvrionmentEOF'
[29422 ms] 
[29428 ms] 
[29430 ms] Start: Run in container: test -f /var/vscode-server-insiders/.patchEtcProfileMarker
[29438 ms] 
[29439 ms] 
[29439 ms] Exit code 1
[29440 ms] Start: Run in container: test ! -f '/var/vscode-server-insiders/.patchEtcProfileMarker' && set -o noclobber && mkdir -p '/var/vscode-server-insiders' && { > '/var/vscode-server-insiders/.patchEtcProfileMarker' ; } 2> /dev/null
[29489 ms] 
[29489 ms] 
[29490 ms] Start: Run in container: sed -i -E 's/((^|\s)PATH=)([^\$]*)$/\1${PATH:-\3}/g' /etc/profile || true
[29508 ms] 
[29508 ms] 
[29513 ms] Start: Run in container: test ! -f '/home/vscode/.vscode-server-insiders/data/Machine/.writeMachineSettingsMarker' && set -o noclobber && mkdir -p '/home/vscode/.vscode-server-insiders/data/Machine' && { > '/home/vscode/.vscode-server-insiders/data/Machine/.writeMachineSettingsMarker' ; } 2> /dev/null
[29524 ms] 
[29525 ms] 
[29525 ms] Start: Run in container: mkdir -p '/home/vscode/.vscode-server-insiders/data/Machine' && cat >'/home/vscode/.vscode-server-insiders/data/Machine/settings.json' <<'settingsJSON'
[29535 ms] 
[29536 ms] 
[29537 ms] Start: Run in container: test -d /home/vscode/.vscode-server-insiders/bin/d624242076d166bb8ce8b314e8d44b8af6100a3d-insider
[29551 ms] 
[29552 ms] 
[29552 ms] Exit code 1
[29553 ms] Installing VS Code Server for commit d624242076d166bb8ce8b314e8d44b8af6100a3d
[29555 ms] Start: Downloading VS Code Server
[29555 ms] d624242076d166bb8ce8b314e8d44b8af6100a3d x64 insider 
[37237 ms] Start: Installing VS Code Server
[37238 ms] Start: Run in container: mkdir -p /home/vscode/.vscode-server-insiders/bin/d624242076d166bb8ce8b314e8d44b8af6100a3d-insider_1626714249020
[37251 ms] 
[37252 ms] 
[37373 ms] Start: Run in container: (dd iflag=fullblock bs=8192 count=6002 2>/dev/null; dd iflag=fullblock bs=3537 count=1 2>/dev/null) | tar --no-same-owner -xz --strip-components 1 -C /home/vscode/.vscode-server-insiders/bin/d624242076d166bb8ce8b314e8d44b8af6100a3d-insider_1626714249020
[38967 ms] 
[38967 ms] 
[38968 ms] Start: Run in container: mv -n /home/vscode/.vscode-server-insiders/bin/d624242076d166bb8ce8b314e8d44b8af6100a3d-insider_1626714249020 /home/vscode/.vscode-server-insiders/bin/d624242076d166bb8ce8b314e8d44b8af6100a3d-insider
[38978 ms] 
[38979 ms] 
[38980 ms] Start: Launching Remote-Containers helper.
[38981 ms] Start: Run: gpgconf --list-dir agent-extra-socket
[39024 ms] findLocalWindowsExecutable: Exectuable 'gpgconf' not found on PATH 'C:\WINDOWS\system32;C:\WINDOWS;C:\WINDOWS\System32\Wbem;C:\WINDOWS\System32\WindowsPowerShell\v1.0\;C:\WINDOWS\System32\OpenSSH\;C:\Program Files\dotnet\;C:\Program Files\Microsoft SQL Server\130\Tools\Binn\;C:\Program Files\Microsoft SQL Server\Client SDK\ODBC\170\Tools\Binn\;C:\Program Files\TortoiseGit\bin;C:\PROGRA~1\JPKI;C:\Program Files (x86)\Common Files\Sony Shared\FeliCaLibrary;C:\Program Files\Common Files\Sony Shared\FeliCaLibrary;C:\Program Files (x86)\Common Files\Sony Shared\FeliCaNFCLibrary;C:\Program Files\Common Files\Sony Shared\FeliCaNFCLibrary;C:\Program Files\Git\cmd;C:\Program Files\Docker\Docker\resources\bin;C:\ProgramData\DockerDesktop\version-bin;C:\Users\juner\AppData\Local\Microsoft\WindowsApps;C:\Users\juner\.dotnet\tools;C:\Users\juner\AppData\Local\Programs\Microsoft VS Code\bin;;C:\Users\juner\AppData\Local\Programs\Microsoft VS Code Insiders\bin'.
[39033 ms] userEnvProbe: loginInteractiveShell (default)
[39035 ms] userEnvProbe shell: /bin/bash
[39036 ms] Start: Run in container: command -v git >/dev/null 2>&1 && git config --system credential.helper '!f() { /home/vscode/.vscode-server-insiders/bin/d624242076d166bb8ce8b314e8d44b8af6100a3d-insider/node /tmp/vscode-remote-containers-2c84720fd998aeb72f4eb011523f92673b045bff.js $*; }; f' || true
[39074 ms] Start: Run in container: cat <<'EOF-/tmp/vscode-remote-containers-2c84720fd998aeb72f4eb011523f92673b045bff.js' >/tmp/vscode-remote-containers-2c84720fd998aeb72f4eb011523f92673b045bff.js
[39087 ms] 
[39087 ms] 
[39100 ms] Start: Run in container: cat <<'EOF-/tmp/vscode-remote-containers-server-2c84720fd998aeb72f4eb011523f92673b045bff.js' >/tmp/vscode-remote-containers-server-2c84720fd998aeb72f4eb011523f92673b045bff.js
[39107 ms] 
[39107 ms] 
[39108 ms] Start: Run in container: /home/vscode/.vscode-server-insiders/bin/d624242076d166bb8ce8b314e8d44b8af6100a3d-insider/node /tmp/vscode-remote-containers-server-2c84720fd998aeb72f4eb011523f92673b045bff.js
[39165 ms] 
[39165 ms] 
[39166 ms] Start: Preparing Extensions
[39166 ms] Start: Run in container: test ! -f '/home/vscode/.vscode-server-insiders/data/Machine/.installExtensionsMarker' && set -o noclobber && mkdir -p '/home/vscode/.vscode-server-insiders/data/Machine' && { > '/home/vscode/.vscode-server-insiders/data/Machine/.installExtensionsMarker' ; } 2> /dev/null
[39174 ms] 
[39174 ms] 
[39180 ms] Extensions cache, install extensions: felixfbecker.php-debug, bmewburn.vscode-intelephense-client, mrmlnc.vscode-apache, MS-CEINTL.vscode-language-pack-ja
[39181 ms] Start: Run in container: test -d /home/vscode/.vscode-server-insiders/extensionsCache && ls /home/vscode/.vscode-server-insiders/extensionsCache || true
[39186 ms] 
[39187 ms] 
[39190 ms] Extensions cache, copy to remote: None
[39191 ms] Start: Run in container: for pid in `cd /proc && ls -d [0-9]*`; do { echo $pid ; readlink /proc/$pid/cwd ; readlink /proc/$pid/ns/mnt ; cat /proc/$pid/stat | tr "
[39320 ms] Start: Starting VS Code Server
[39321 ms] Start: Run in container: /home/vscode/.vscode-server-insiders/bin/d624242076d166bb8ce8b314e8d44b8af6100a3d-insider/server.sh --log debug --force-disable-user-env --use-host-proxy --port 0 --extensions-download-dir /home/vscode/.vscode-server-insiders/extensionsCache --install-extension felixfbecker.php-debug --install-extension bmewburn.vscode-intelephense-client --install-extension mrmlnc.vscode-apache --install-extension MS-CEINTL.vscode-language-pack-ja --start-server --disable-websocket-compression
[40738 ms] 

*
* Visual Studio Code Server
*
* Reminder: You may only use this software with Visual Studio family products,
* as described in the license https://aka.ms/vscode-remote/license
*

Extension host agent listening on 40503

[40740 ms] Start: Run in container: echo 40503 >'/home/vscode/.vscode-server-insiders/data/Machine/.devport-d624242076d166bb8ce8b314e8d44b8af6100a3d-insider'
[40750 ms] 
[40751 ms] 
[40752 ms] Port forwarding for container port 40503 starts listening on local port.
[40759 ms] Port forwarding local port 40503 to container port 40503
[40829 ms] Start: Run in container: test ! -f '/home/vscode/.vscode-server-insiders/data/Machine/.onCreateCommandMarker' && set -o noclobber && mkdir -p '/home/vscode/.vscode-server-insiders/data/Machine' && { > '/home/vscode/.vscode-server-insiders/data/Machine/.onCreateCommandMarker' ; } 2> /dev/null
[40839 ms] 
[40840 ms] 
[40841 ms] Start: Run in container: test ! -f '/home/vscode/.vscode-server-insiders/data/Machine/.updateContentCommandMarker' && set -o noclobber && mkdir -p '/home/vscode/.vscode-server-insiders/data/Machine' && { > '/home/vscode/.vscode-server-insiders/data/Machine/.updateContentCommandMarker' ; } 2> /dev/null
[40853 ms] 
[40854 ms] 
[40855 ms] Start: Run in container: test ! -f '/home/vscode/.vscode-server-insiders/data/Machine/.postCreateCommandMarker' && set -o noclobber && mkdir -p '/home/vscode/.vscode-server-insiders/data/Machine' && { > '/home/vscode/.vscode-server-insiders/data/Machine/.postCreateCommandMarker' ; } 2> /dev/null
[40868 ms] 
[40869 ms] 
[40905 ms] Start: Run in container: # Test for /home/vscode/.gitconfig and git
[40922 ms] 
[40923 ms] 
[40923 ms] Start: Run in container: # Copy C:\Users\juner\.gitconfig to /home/vscode/.gitconfig
[40944 ms] 
[40947 ms] 
[40950 ms] Start: Run in container: command -v git >/dev/null 2>&1 && git config --global credential.helper '!f() { /home/vscode/.vscode-server-insiders/bin/d624242076d166bb8ce8b314e8d44b8af6100a3d-insider/node /tmp/vscode-remote-containers-2c84720fd998aeb72f4eb011523f92673b045bff.js $*; }; f' || true
[41226 ms] 
[41227 ms] 
[41228 ms] Start: Run in container: for pid in `cd /proc && ls -d [0-9]*`; do { echo $pid ; readlink /proc/$pid/cwd ; readlink /proc/$pid/ns/mnt ; cat /proc/$pid/stat | tr "
[41543 ms] userEnvProbe PATHs:
Probe:     '/usr/local/share/nvm/current/bin:/usr/local/share/nvm/versions/node/v14.17.3/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/home/vscode/.local/bin'
Container: '/usr/local/share/nvm/current/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin'
[41585 ms] Start: Run in container: mkdir -p '/home/vscode/.vscode-server-insiders/data/Machine' && [ "$(cat '/home/vscode/.vscode-server-insiders/data/Machine/.postStartCommandMarker' 2>/dev/null)" != '2021-07-19T17:03:57.031902682Z' ] && echo '2021-07-19T17:03:57.031902682Z' > '/home/vscode/.vscode-server-insiders/data/Machine/.postStartCommandMarker'
[41588 ms] userEnvProbe taking longer than 2 seconds. Process not found.
[41603 ms] 
[41604 ms] 
[41634 ms] Port forwarding connection from 51430 > 40503 > 40503 in the container.
[41636 ms] Start: Run in container: /home/vscode/.vscode-server-insiders/bin/d624242076d166bb8ce8b314e8d44b8af6100a3d-insider/node -e 
[41650 ms] [17:04:13] Extension host agent started.
[41740 ms] Installing extensions...
[42902 ms] Installing extension 'bmewburn.vscode-intelephense-client'...
[42992 ms] Installing extension 'mrmlnc.vscode-apache'...
[43076 ms] Installing extension 'felixfbecker.php-debug'...
[43092 ms] Installing extension 'ms-ceintl.vscode-language-pack-ja'...
[43359 ms] [17:04:15] Installing extension: bmewburn.vscode-intelephense-client
[43539 ms] [17:04:15] Installing extension: mrmlnc.vscode-apache
[43552 ms] [17:04:15] Installing extension: ms-ceintl.vscode-language-pack-ja
[43562 ms] [17:04:15] Installing extension: felixfbecker.php-debug
[45088 ms] Port forwarding connection from 64963 > 40503 > 40503 in the container.
[45089 ms] Start: Run in container: /home/vscode/.vscode-server-insiders/bin/d624242076d166bb8ce8b314e8d44b8af6100a3d-insider/node -e 
[45307 ms] [17:04:17] [::ffff:127.0.0.1][7ae857fe][ManagementConnection] New connection established.
[47128 ms] [17:04:18] Downloaded extension: mrmlnc.vscode-apache /home/vscode/.vscode-server-insiders/extensionsCache/mrmlnc.vscode-apache-1.2.0
[47576 ms] [17:04:19] Extracted extension to /home/vscode/.vscode-server-insiders/extensions/.a427ff70-41fb-4781-a7db-99e8b6a8b7fc: mrmlnc.vscode-apache
[47641 ms] [17:04:19] Renamed to /home/vscode/.vscode-server-insiders/extensions/mrmlnc.vscode-apache-1.2.0
[47642 ms] [17:04:19] Extracting completed. mrmlnc.vscode-apache
[47689 ms] [17:04:19] Extension installed successfully: mrmlnc.vscode-apache
Extension 'mrmlnc.vscode-apache' v1.2.0 was successfully installed.
[47769 ms] [17:04:19] Downloaded extension: ms-ceintl.vscode-language-pack-ja /home/vscode/.vscode-server-insiders/extensionsCache/ms-ceintl.vscode-language-pack-ja-1.58.8
[48020 ms] [17:04:19] Downloaded extension: felixfbecker.php-debug /home/vscode/.vscode-server-insiders/extensionsCache/felixfbecker.php-debug-1.16.1
[48298 ms] [17:04:20] Downloaded extension: bmewburn.vscode-intelephense-client /home/vscode/.vscode-server-insiders/extensionsCache/bmewburn.vscode-intelephense-client-1.7.1
[49203 ms] Extensions cache, remote removals: None
[49205 ms] Start: Run in container: tar c extensionsCache/bmewburn.vscode-intelephense-client-1.7.1 extensionsCache/felixfbecker.php-debug-1.16.1 extensionsCache/mrmlnc.vscode-apache-1.2.0 extensionsCache/ms-ceintl.vscode-language-pack-ja-1.58.8
[49452 ms] [17:04:21] [::ffff:127.0.0.1][70d76c68][ExtensionHostConnection] New connection established.
[49471 ms] [17:04:21] [::ffff:127.0.0.1][70d76c68][ExtensionHostConnection] <498> Launched Extension Host Process.
[52136 ms] [17:04:23] Extracted extension to /home/vscode/.vscode-server-insiders/extensions/.04ff8412-0888-4c6e-ab92-510f22758c53: ms-ceintl.vscode-language-pack-ja
[52236 ms] [17:04:24] Renamed to /home/vscode/.vscode-server-insiders/extensions/ms-ceintl.vscode-language-pack-ja-1.58.8
[52264 ms] [17:04:24] Extracting completed. ms-ceintl.vscode-language-pack-ja
[52335 ms] [17:04:24] Adding language packs from the extension ms-ceintl.vscode-language-pack-ja
[52801 ms] [17:04:24] Extension installed successfully: ms-ceintl.vscode-language-pack-ja
[52805 ms] Extension 'ms-ceintl.vscode-language-pack-ja' v1.58.8 was successfully installed.
[56275 ms] [17:04:28] Extracted extension to /home/vscode/.vscode-server-insiders/extensions/.538f920c-7743-4e2a-8ff6-f655e99a153a: bmewburn.vscode-intelephense-client
[56305 ms] [17:04:28] Renamed to /home/vscode/.vscode-server-insiders/extensions/bmewburn.vscode-intelephense-client-1.7.1
[56312 ms] [17:04:28] Extracting completed. bmewburn.vscode-intelephense-client
[56328 ms] [17:04:28] Extension installed successfully: bmewburn.vscode-intelephense-client
[56330 ms] Extension 'bmewburn.vscode-intelephense-client' v1.7.1 was successfully installed.
[56885 ms] [17:04:28] Extracted extension to /home/vscode/.vscode-server-insiders/extensions/.0cb817d6-9b17-4b2a-90dd-a495e2b6f09e: felixfbecker.php-debug
[56899 ms] [17:04:28] Renamed to /home/vscode/.vscode-server-insiders/extensions/felixfbecker.php-debug-1.16.1
[56902 ms] [17:04:28] Extracting completed. felixfbecker.php-debug
[56914 ms] [17:04:28] Extension installed successfully: felixfbecker.php-debug
[56916 ms] Extension 'felixfbecker.php-debug' v1.16.1 was successfully installed.
[112720 ms] Extensions cache, remote removals: None
```

</details>

**Comment by juner** [comment]
error. not work.

<details><summary>log</summary>

```
[2021-07-20T03:30:08.723Z] Remote-Containers 0.188.2 in VS Code 1.59.0-insider (d624242076d166bb8ce8b314e8d44b8af6100a3d).
[2021-07-20T03:30:08.723Z] Start: Resolving Remote
[2021-07-20T03:30:08.726Z] Setting up container for folder or workspace: c:\*************\source\repos\github\errord_devcontainer20210709
[2021-07-20T03:30:08.728Z] Start: Check Docker is running
[2021-07-20T03:30:08.728Z] Start: Run: docker version --format {{.Server.APIVersion}}
[2021-07-20T03:30:09.226Z] Stop (498 ms): Run: docker version --format {{.Server.APIVersion}}
[2021-07-20T03:30:09.227Z] Server API version: 1.41
[2021-07-20T03:30:09.227Z] Stop (499 ms): Check Docker is running
[2021-07-20T03:30:09.337Z] Start: Run: docker-compose version --short
[2021-07-20T03:30:10.426Z] Stop (1089 ms): Run: docker-compose version --short
[2021-07-20T03:30:10.427Z] Start: Run: docker ps -q -a --filter label=com.docker.compose.project=errord_devcontainer20210709 --filter label=com.docker.compose.service=php
[2021-07-20T03:30:10.935Z] Stop (508 ms): Run: docker ps -q -a --filter label=com.docker.compose.project=errord_devcontainer20210709 --filter label=com.docker.compose.service=php
[2021-07-20T03:30:10.936Z] Start: Run: docker-compose -f c:\*************\source\repos\github\errord_devcontainer20210709\docker-compose.yml config
[2021-07-20T03:30:12.172Z] Stop (1236 ms): Run: docker-compose -f c:\*************\source\repos\github\errord_devcontainer20210709\docker-compose.yml config
[2021-07-20T03:30:12.172Z] 
[2021-07-20T03:30:12.173Z] Traceback (most recent call last):
  File "docker-compose", line 3, in <module>
  File "compose\cli\main.py", line 81, in main
  File "compose\cli\main.py", line 197, in perform_command
  File "compose\metrics\decorator.py", line 18, in wrapper
  File "compose\cli\main.py", line 441, in config
UnicodeEncodeError: 'cp932' codec can't encode character '\U0001f31f' in position 259: illegal multibyte sequence
[5844] Failed to execute script docker-compose

[2021-07-20T03:30:12.173Z] Exit code 4294967295
[2021-07-20T03:30:12.173Z] Start: Run: docker-compose -f c:\*************\source\repos\github\errord_devcontainer20210709\docker-compose.yml config
[2021-07-20T03:30:12.379Z] 
[2021-07-20T03:30:13.598Z] services:
  php:
    build:
      args:
        INSTALL_NODE: "true"
        NODE_VERSION: lts/*
        VARIANT: '7'
      context: c:\*************\source\repos\github\errord_devcontainer20210709\
.container
    environment:
      APP_TITLE: ＰＨＰサンプル�
    volumes:
    - c:\*************\source\repos\github\errord_devcontainer20210709:/var/www/
html:rw
version: '3.7'

[2021-07-20T03:30:13.802Z] Stop (1629 ms): Run: docker-compose -f c:\*************\source\repos\github\errord_devcontainer20210709\docker-compose.yml config
[2021-07-20T03:30:13.807Z] YAMLException: can not read a block mapping entry; a multiline key may not be an implicit key (10:16)

  7 |         VARIANT: '7'
  8 |       context: c:\*************\source\repos\g ...
  9 | .container
 10 |     environment:
---------------------^
 11 |       APP_TITLE: ＰＨＰサンプル�
 12 |     volumes:
	at tL (c:\*************\.vscode-insiders\extensions\ms-vscode-remote.remote-containers-0.188.2\dist\extension\extension.js:193:938)
	at K (c:\*************\.vscode-insiders\extensions\ms-vscode-remote.remote-containers-0.188.2\dist\extension\extension.js:193:972)
	at kJ (c:\*************\.vscode-insiders\extensions\ms-vscode-remote.remote-containers-0.188.2\dist\extension\extension.js:200:1955)
	at sc (c:\*************\.vscode-insiders\extensions\ms-vscode-remote.remote-containers-0.188.2\dist\extension\extension.js:200:4873)
	at FJ (c:\*************\.vscode-insiders\extensions\ms-vscode-remote.remote-containers-0.188.2\dist\extension\extension.js:200:7250)
	at oL (c:\*************\.vscode-insiders\extensions\ms-vscode-remote.remote-containers-0.188.2\dist\extension\extension.js:201:258)
	at MJ (c:\*************\.vscode-insiders\extensions\ms-vscode-remote.remote-containers-0.188.2\dist\extension\extension.js:201:478)
	at Fte (c:\*************\.vscode-insiders\extensions\ms-vscode-remote.remote-containers-0.188.2\dist\extension\extension.js:254:1075)
	at async Nte (c:\*************\.vscode-insiders\extensions\ms-vscode-remote.remote-containers-0.188.2\dist\extension\extension.js:236:2092)
	at async BF (c:\*************\.vscode-insiders\extensions\ms-vscode-remote.remote-containers-0.188.2\dist\extension\extension.js:236:453)
	at async Bte (c:\*************\.vscode-insiders\extensions\ms-vscode-remote.remote-containers-0.188.2\dist\extension\extension.js:254:10674)

```
</details>

**Comment by chrmarti** [comment]
@juner Thanks, I have updated the fix and am releasing 0.188.3 for VS Code Insiders.

Released for VS Code 1.58 with Remote-Containers 0.187.1.

**Comment by juner** [comment]
thank you very much!
